jsx/App/Stories/Story/Sidebar/Sidebar.jsx

```diff
@@ -1,7 +1,7 @@
 import { Title } from './Title.jsx';
 import { Video } from './Video.jsx';
 import { Minibar } from './Minibar/Minibar.jsx'
-import { getMediaFilePath } from '../Story.jsx';
+import { getMediaFilePath } from '~./jsx/App/Stories/Story/Story.jsx';
```

Comment:
If the path only has a single `../`, I think it's more readable the old way than as an absolute path, so we might want to change it back. Do you agree?

Reply:
SG!